<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgPaths</class>
 <widget class="QDialog" name="CQDlgPaths">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqShowOrientation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show orientation of points</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowPathLine">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show path line</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowPosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show current position on path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Line size [pt]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqLineSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Size of ctrl points [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqControlPointSize">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="qqAdjustColor">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust line color</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="qqDistanceCombo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>246</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance unit</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqShowShapingDialog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Show path shaping dialog</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqDistanceCombo</tabstop>
  <tabstop>qqShowShapingDialog</tabstop>
  <tabstop>qqShowOrientation</tabstop>
  <tabstop>qqShowPathLine</tabstop>
  <tabstop>qqShowPosition</tabstop>
  <tabstop>qqAdjustColor</tabstop>
  <tabstop>qqLineSize</tabstop>
  <tabstop>qqControlPointSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
